R/trackhub.R

```diff
@@ -462,8 +462,12 @@ setMethod("track", "TrackHubGenome", function(object, name, ...) {
         if (isEmpty(track[[1]]@bigDataUrl)) {
             stop("Track '", name, "' does not contain any data file")
         }
-        import(paste0(uri(trackhub(object)), "/", track[[1]]@bigDataUrl))
-    }else{
+        if (uriIsWritable(track[[1]]@bigDataUrl)) {
```

Comment:
In track hub `bigDataUrl` can be a local resource on the same track hub or it can be a external resource of another track hub(i.e URL)  so with the `uriIsWritable` it's checking whether resource is internal or external resource.